numba/tests/test_stencils.py

```diff
@@ -1220,8 +1393,11 @@ def generate_stencil_tree(
         # If astor is installed the decompilation of the AST is also printed
         DEBUG = False
         if DEBUG:
-            print("ORIGINAL")
-            print(ast.dump(tree))
+            try:
+                print("ORIGINAL")
+                print(ast.dump(tree))
+            except ImportError:
```

Comment:
Remove this try-except?

Reply:
Done in https://github.com/numba/numba/pull/3719/commits/5dbca1ff3baac88354934043a83d74442b14b26d